votingpool/factory_test.go

```diff
@@ -243,27 +301,29 @@ func TstCreateCreditsOnNewSeries(t *testing.T, pool *Pool, amounts []int64) (uin
 }
 
 // TstCreateSeriesCredits creates a new credit for every item in the amounts
-// slice, locked to the given series' address with branch==1 and index==0.
+// slice, locked to the given series' address with branch==1 and the index
+// varying according to the index of the amount on the amounts slice.
 func TstCreateSeriesCredits(t *testing.T, pool *Pool, seriesID uint32, amounts []int64) []credit {
-	addr := TstNewWithdrawalAddress(t, pool, seriesID, Branch(1), Index(0))
-	pkScript, err := txscript.PayToAddrScript(addr.addr)
-	if err != nil {
-		t.Fatal(err)
-	}
-	msgTx := createMsgTx(pkScript, amounts)
-	txSha := msgTx.TxSha()
 	credits := make([]credit, len(amounts))
-	for i := range msgTx.TxOut {
+	for i, amount := range amounts {
+		addr := TstNewWithdrawalAddress(t, pool, seriesID, Branch(1), Index(i))
+		pkScript, err := txscript.PayToAddrScript(addr.addr)
+		if err != nil {
+			t.Fatal(err)
+		}
+		msgTx := createMsgTx(pkScript, []int64{amount})
+		txSha := msgTx.TxSha()
+		idx := 0
```

Comment:
Oh.. ok I misread that and am confusing inputs with outputs.  It's referencing the one output, since you only passed it one amount.  Why not pass a slice (or varargs) of wire.TxOut instead?